mono/mini/debugger-agent.c

```diff
@@ -6538,7 +6538,8 @@ do_invoke_method (DebuggerTlsData *tls, Buffer *buf, InvokeData *invoke, guint8
 			else {
 				ERROR_DECL (error);
 				this_arg = mono_object_new_checked (domain, m->klass, error);
-				mono_error_assert_ok (error);
+				if (!is_ok (error))
+					return ERR_INVALID_ARGUMENT;
```

Comment:
This is missing a `mono_error_cleanup (error)` on the return branch.  The original PR also needs a followup: https://github.com/mono/mono/pull/19542/files#r415735983

Reply:
Okay, I will fix it.